tdp/components/yarn.yml

```diff
@@ -40,6 +40,10 @@
   depends_on:
     - hadoop_install
 
+- name: yarn_jmx-exporter_config
```

Comment:
This component should be added to `yarn_config` dependencies. If you do not add the dependency, running a DAG to `yarn_config` will not run `yarn_jmx-exporter_config`.

Reply:
ok, I just realized this yarn_config and it's depends on all components config, I'll fix this and also add for hdfs_config

Reply:
Done